terraform/cloudwatch.tf.json.template.py

```diff
@@ -7,28 +7,48 @@
 
 emit_tf(None if config.disable_monitoring else {
     "resource": [
-        {
-            "aws_cloudwatch_metric_alarm": {
-                f"{lambda_}_5xx": {
-                    "alarm_name": config.qualified_resource_name(lambda_ + '_5xx'),
-                    "comparison_operator": "GreaterThanThreshold",
-                    # This alarm will catch persistent 5XX errors occurring over
-                    # one hour, specifically when more than one occurrence is
-                    # sampled in a ten-minute period for six consecutive periods.
-                    "evaluation_periods": 6,
-                    "period": 60 * 10,
-                    "metric_name": "5XXError",
-                    "namespace": "AWS/ApiGateway",
-                    "statistic": "Sum",
-                    "threshold": 1,
-                    "treat_missing_data": "notBreaching",
-                    "dimensions": {
-                        "ApiName": config.qualified_resource_name(lambda_),
-                        "Stage": config.deployment_stage,
+        *(
+            {
+                "aws_cloudwatch_metric_alarm": {
+                    f"{lambda_}_5xx": {
+                        "alarm_name": config.qualified_resource_name(lambda_ + '_5xx'),
+                        "comparison_operator": "GreaterThanThreshold",
+                        # This alarm will catch persistent 5XX errors occurring over
+                        # one hour, specifically when more than one occurrence is
+                        # sampled in a ten-minute period for six consecutive periods.
+                        "evaluation_periods": 6,
+                        "period": 60 * 10,
+                        "metric_name": "5XXError",
+                        "namespace": "AWS/ApiGateway",
+                        "statistic": "Sum",
+                        "threshold": 1,
+                        "treat_missing_data": "notBreaching",
+                        "dimensions": {
+                            "ApiName": config.qualified_resource_name(lambda_),
+                            "Stage": config.deployment_stage,
+                        },
+                        "alarm_actions": ["${aws_sns_topic.azul_monitoring.arn}"]
                     }
                 }
             }
+            for lambda_ in config.lambda_names()
+        ),
+        {
+            'aws_sns_topic': {
+                'azul_monitoring': {
+                    'name': 'azul-monitoring'
+                }
+            },
+            'aws_sns_topic_subscription': {
+                'azul_monitoring_azul_group': {
+                    'topic_arn': '${aws_sns_topic.azul_monitoring.arn}',
+                    # The `email` protocol is only partially supported. Be careful
+                    # when changing or removing this.
+                    # https://registry.terraform.io/providers/hashicorp/aws/4.3.0/docs/resources/sns_topic_subscription#protocol-support
```

Comment:
Too handwavy. After reading the docs, it appears to me that there is something to be done after `make deploy`. That step needs to be documented or automated. Think about how you can make that as convenient and automatic as possible.